_ml-commons-plugin/connectors.md

```diff
@@ -0,0 +1,392 @@
+---
+layout: default
+title: Connecting to other ML platforms
+has_children: false
+nav_order: 60
+---
+
+# Connecting to other ML platforms
```

Comment:
Maybe "Connecting to third-party ML platforms"? Otherwise, it's not clear what "other" refers to.

Reply:
```suggestion
# Connecting to third-party ML platforms
```